src/MainContent.tsx

```diff
@@ -325,6 +338,9 @@ function MainContent({ projectConfig, updateToken: setAuthToken }: MainContentPr
         loadDocument();
     }, [fetchedDocument]);
 
+    useEffect(() => {
+        zoomToPage();
+    }, [multiLayoutMode, zoomToPage]);
```

Comment:
only if multilayoutmode is set to false

Reply:
Correct, this is set in the `zoomToPage` function line `124`

Reply:
From a code cleanness this isn't very clean, zoomToPage indicates a function that should always zoom to page when triggered, please move the check before triggering it

Reply:
done cb480e546f6698f0b3716738fab348e3e8edd31a